djstripe/templates/djstripe/email/body_base.txt

```diff
@@ -13,7 +13,7 @@ DETAILS
 -------
 {{ charge.customer.subscription.plan.name }} - ${{ charge.amount|floatformat:2 }}
 
-TOTAL: ${{ charge.amount|floatformat:2 }} USD
+TOTAL: ${{ charge.amount|floatformat:2 }} {{ charge.currency|upper }}
```

Comment:
@kavdev That's a good point! Removing it would be better for non-dollar-based currencies. It made sense for me here in Canada, but it's a big world out there.